homework3/semi-uber/Server/api/Schemas/dimensions.schema.js

```diff
@@ -0,0 +1,8 @@
+const Joi = require("@hapi/joi");
+const dimensionsSchema = Joi.object({
+  width: Joi.number().required(),
```

Comment:
Add check for this numbers to be positive as well

Reply:
done